src/cli/compiler.js

```diff
@@ -66,6 +67,80 @@ class Compiler {
     }
   }
 
+  getFileLocations() {
+    try {
+      let allFiles = []
+      let subgraph = this.loadSubgraph()
+
+      // Add all file paths specified in manifest
+      allFiles.push(subgraph.getIn(['schema', 'file']))
+      let mappingFiles = subgraph.get('dataSources').map(dataSource => {
```

Comment:
Drop the `let mappingFiles = ` here, since we're not using `mappingFiles` anywhere.